### ISSUE TITLE
Serious problem encountered when trying to downgrad firmware from 1.1.1 back to 1.0.2

### ISSUE THREAD

**Issue by GoogleCodeExporter** [San112/iphone-elite#7]
```
Hi everyone, I need some major help regarding downgrading from 1.1.1 to 1.0.2 
(just firmware) 
here. I think I encountered a rare problem here. I'm not perfectly sure if this 
is the place to post 
this but would like to present it anyway.

I have an iPhone (1.0.2 OTB). It was activated and jailbroken with iNdependence 
1.2.2 but not 
unlocked for phone functions. I used it for a week until one day when I tried 
to install apptapp 
but chose the wrong firmware (I chose 1.0.0 instead of 1.0.2). And I got the 
yellow triangle 
screen. Not knowing how to fix this, I risked to upgrade it to 1.1.1 with the 
hope of downgrading 
to it's original 1.0.2 status again for activation.

But ever since that I haven't been able to restore it to 1.0.2. I restored it a 
hundred times to 1.1.1 
and tried to downgrade it to 1.0.2 so that I could at least use iNdependence to 
activate it. I tried 
various methods entering DFU with iBrickr, iNdependence and manually and 
shift-selecting 1.0.2 
fw to restore, on Windows XP and MAC, under iTunes 7.4.2 and 7.4.1. 

But I keep getting the 1.1.1 usb cable and itunes icon screen, which means it's 
still with 1.1.1. So 
it becomes impossible to re-activate under 1.0.2 and there's no method 
available to activate and 
unlock a 1.1.1 that's activated and upgraded from 1.0.2 at this moment.

I at least would be very happy if I got to use the other functions. It's now 
basically useless. I 
started a thread elsewhere but haven't been able to fix it with the suggestions 
posted. Please 
guide me here. Thank you so much!

```

Original issue reported on code.google.com by `kin....@gmail.com` on 23 Oct 2007 at 4:56

Attachments:
- [](https://storage.googleapis.com/google-code-attachments/iphone-elite/issue-7/comment-0/)
